application/performance/locust/files/ccs_1500_destinations.json

```diff
@@ -0,0 +1 @@
7436}, {"reference": "destination1464", "latitude": 51.517478, "longitude": -0.437208}, {"reference": "destination1465", "latitude": 51.577482, "longitude": -1.834822}, {"reference": "destination1466", "latitude": 51.688163, "longitude": -1.275136}, {"reference": "destination1467", "latitude": 51.338837, "longitude": -0.746153}, {"reference": "destination1468", "latitude": 51.885688, "longitude": -0.428408}, {"reference": "destination1469", "latitude": 51.452539, "longitude": -2.126709}, {"reference": "destination1470", "latitude": 51.547086, "longitude": -0.482006}, {"reference": "destination1471", "latitude": 51.426252, "longitude": -0.73138}, {"reference": "destination1472", "latitude": 51.637972, "longitude": -2.16075}, {"reference": "destination1473", "latitude": 51.730813, "longitude": -0.458381}, {"reference": "destination1474", "latitude": 51.671061, "longitude": -1.281049}, {"reference": "destination1475", "latitude": 52.000511, "longitude": -0.987513}, {"reference": "destination1476", "latitude": 51.819593, "longitude": -0.822621}, {"reference": "destination1477", "latitude": 51.526077, "longitude": -0.461175}, {"reference": "destination1478", "latitude": 51.561288, "longitude": -1.760764}, {"reference": "destination1479", "latitude": 52.087489, "longitude": -1.285838}, {"reference": "destination1480", "latitude": 51.819209, "longitude": -0.813207}, {"reference": "destination1481", "latitude": 51.410195, "longitude": -0.845603}, {"reference": "destination1482", "latitude": 52.051501, "longitude": -1.78283}, {"reference": "destination1483", "latitude": 52.025437, "longitude": -0.737215}, {"reference": "destination1484", "latitude": 51.639612, "longitude": -1.507353}, {"reference": "destination1485", "latitude": 52.214169, "longitude": -1.870616}, {"reference": "destination1486", "latitude": 51.258269, "longitude": -0.542178}, {"reference": "destination1487", "latitude": 51.600599, "longitude": -1.124189}, {"reference": "destination1488", "latitude": 51.333905, "longitude": -0.744969}, {"reference": "destination1489", "latitude": 52.143512, "longitude": -0.455796}, {"reference": "destination1490", "latitude": 51.919123, "longitude": -0.664652}, {"reference": "destination1491", "latitude": 51.432461, "longitude": -0.954127}, {"reference": "destination1492", "latitude": 51.637178, "longitude": -0.466998}, {"reference": "destination1493", "latitude": 52.008904, "longitude": -0.773208}, {"reference": "destination1494", "latitude": 52.010018, "longitude": -0.510931}, {"reference": "destination1495", "latitude": 51.72736, "longitude": -1.202042}, {"reference": "destination1496", "latitude": 51.509592, "longitude": -0.640013}, {"reference": "destination1497", "latitude": 51.400749, "longitude": -0.759831}, {"reference": "destination1498", "latitude": 51.885662, "longitude": -0.519311}, {"reference": "destination1499", "latitude": 51.415329, "longitude": -0.750777}, {"reference": "destination1500", "latitude": 52.214996, "longitude": -1.870365}]}
```

Comment:
Why is this being recreated - are the files in the mock folder not reusable for the testing for locust?

Reply:
These files can all be deleted as the config file already points to the mock folder: ccs_prefix